sdk/evaluation/azure-ai-evaluation/CHANGELOG.md

```diff
@@ -44,13 +47,23 @@ outputs = asyncio.run(custom_simulator(
 
 ### Other Changes
 - Improved error messages for the `evaluate` API by enhancing the validation of input parameters. This update provides more detailed and actionable error descriptions.
+- `GroundednessEvaluator` now supports `query` as an optional input in single-turn evaluation. If `query` is provided, a different prompt template will be used for the evaluation.
 - To align with our support of a diverse set of models, the following evaluators will now have a new key in their result output without the `gpt_` prefix. To maintain backwards compatibility, the old key with the `gpt_` prefix will still be present in the output; however, it is recommended to use the new key moving forward as the old key will be deprecated in the future.
   - `CoherenceEvaluator`
   - `RelevanceEvaluator`
   - `FluencyEvaluator`
   - `GroundednessEvaluator`
   - `SimilarityEvaluator`
   - `RetrievalEvaluator`
+- The following evaluators will now have a new key in their result output including LLM reasoning behind the score. The new key will follow the pattern "<metric_name>_reason". The reasoning is the result of a more detailed prompt template being used to generate the LLM response. Note that this requires the maximum number of tokens used to run these evaluators to be increased.
```

Comment:
How about "including chain-of-thought LLM reasoning behind the score""?

Reply:
The long chain of thought isn't being output in the `reason` attribute though - only the summarized LLM reasoning is.

For example, this is what an LLM output would look like:

"<S0>Let's think step by step: The response 'Honolulu' is a single word. "
  "It does not form a complete sentence, lacks grammatical structure, and does not "
  "convey any clear idea or message. It is not possible to assess vocabulary range, "
  "sentence complexity, coherence, or overall readability from a single word. Therefore,"
  "it falls into the category of minimal command of the language.</S0>"
  "<S1>The response is a single word and does not provide any meaningful content to evaluate"
  " fluency. It is largely incomprehensible and does not meet the criteria for higher fluency "
  "levels.</S1><S2>1</S2>"
  
  `reason` attribute is the section between S1s and `score` attribute is the value between S2s

Reply:
Yes, I am aware. And I think we can leave it as is.